src/main/scala/bot/MessageFormatter.scala

```diff
@@ -12,12 +12,14 @@ class MessageFormatter {
       case (_, firstName, _) => firstName
     }
     s"""
-      |*$name*, welcome to Cavisto bot.
-      |
-      |_Available commands:_
-      |/advise -- Ask for wine advice, using filters
-      |/search -- Search wines by name. Example: /search cabernet
-      |""".stripMargin
+       |*$name*, welcome to Cavisto bot.
+       |
+       |_Available commands:_
+       |/start — Get a help message.
+       |/advise — Ask for wine advice, using filters.
+       |/search ... — Search wines by name. Example: /search cabernet
+       |/note — Create a note about wine.
```

Comment:
A "/help" command would probably be good, if the user will need a reminder of what commands you have.